convert_state.py

```diff
@@ -1,60 +1,65 @@
+"""This code converts written state names to the 2 letter abbreviations, or the reverse"""
+import json
 
-
-
-import json  # import json for json processing
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """
+    reads in the data from state_data json file
+    """
+    with open('state_data.json','r') as file:
```

Comment:
with open('state_data.json', 'r') as file:

Reply:
I'm not sure what you mean here, I might be too sleepy, they look the same to me

Reply:
oh i should have noted what I did lol.  I just added a space before the 'r', it's more proper.